options/comments.md

```diff
@@ -93,8 +93,27 @@ Prevents warnings due to unspecified tags from being reported in comments within
 }
 ```
 
-Override TypeDoc's supported block tags, emit warnings for any tags not listed here.
-This option will be set by `tsdoc.json` if present.
+This specifics all of the [JSDoc block tags](../guides/tags.md#block-tags) that TypeDoc considers to be valid.
```

Comment:
I have been intentionally avoiding using "JSDoc" in any of the documentation. JSDoc is dead. Anyone who tries to use the JSDoc documentation to determine how TypeDoc works is in a world of hurt.